src/lib.rs

```diff
@@ -91,11 +91,8 @@
 //!
 //! ### Simple types
 //!
-//! ```
-//! # // Import macros if derive feature is not used.
-//! # #[cfg(not(feature="derive"))]
-//! # use parity_scale_codec_derive::{Encode, Decode};
-//!
+#![cfg_attr(feature = "derive", doc = "```rust")]
+#![cfg_attr(not(feature = "derive"), doc = "*(Only compiled with feature `derive`)*\n```ignore")]
```

Comment:
Is the part in `*(...)*` a comment? Haven't seen this syntax before.

Reply:
Just italic with `()`, renders as such:

![image](https://user-images.githubusercontent.com/10380170/235968322-21f0c26b-97c6-40a0-8660-800c1056d4b2.png)

Reply:
Oh right, had a brainfart. 🙈 Thanks for the responses, looks good!